src/main/java/service/impl/FruitTransactionParserImpl.java

```diff
@@ -0,0 +1,20 @@
+package service.impl;

```

Comment:
The package declaration should not be followed by an empty line.

Reply:
it doesn't even pass checkstyle!